.ado/jobs/integration-test.yml

```diff
@@ -37,11 +37,16 @@ parameters:
             BuildConfiguration: Release
             DeployOptions: --deploy-from-layout # We avoid certs for AppX generation
             UseHermes: true
-          - Name: X86Debug
+          - Name: X86WebDebug
             BuildPlatform: x86
             BuildConfiguration: Debug
             DeployOptions:
             UseHermes: false
+          - Name: X86ReleaseHermes
```

Comment:
smallest of nitpick: Other names are <Platform><Type><Configuration>. This is <Platform><Configuration><Type>

Reply:
Did a quick survey of existing names. It seems like the `WebDebug` ones are the odd out. I think we could do `DebugWeb` to signify `Web` as the more specific flavor, but also `Web Debugging` is the more common phrase, so it seems like it might be more immediately intuitive.